#Request
string modality
string second_information
geometry_msgs/Pose target_pose
float32[] target_joints
---
#Response
bool aruco_found
float32 x
float32 y
float32 z
float32[] vector
int32[] dim#dim[0]=rows; dim[1]=columns
int32 moreTargets
